docs/Tutorial/getting-started.md

```diff
@@ -0,0 +1,128 @@
+                                                   'operation': 'Update',
+                                                   'time': datetime.datetime(2022, 12, 12, 16, 41, 45, tzinfo=tzlocal())}],
+                               'name': 'hello-kubecon-ingress',
+                               'namespace': 'ing-test',
+                               'owner_references': None,
+                               'resource_version': '9123',
+                               'self_link': None,
+                               'uid': '9dda237a-d903-4031-947d-3a5fd9b4d34e'},
+                  'spec': {'default_backend': None,
+                           'ingress_class_name': 'public',
+                           'rules': [{'host': 'hello-kubecon',
+                                      'http': {'paths': [{'backend': {'resource': None,
+                                                                      'service': {'name': 'hello-kubecon-service',
+                                                                                  'port': {'name': None,
+                                                                                           'number': 8080}}},
+                                                          'path': '/',
+                                                          'path_type': 'Prefix'}]}}],
+                           'tls': None},
+                  'status': {'load_balancer': {'ingress': [{'hostname': None,
+                                                            'ip': '127.0.0.1'}]}}}],
+       'kind': 'IngressList',
+       'metadata': {'_continue': None,
+                    'remaining_item_count': None,
+                    'resource_version': '9187',
+                    'self_link': None}}
+  status: completed
+  timing:
+    completed: 2022-12-12 16:42:11 +0000 UTC
+    enqueued: 2022-12-12 16:42:08 +0000 UTC
+    started: 2022-12-12 16:42:11 +0000 UTC
+```
+This shows us a number of things including the name of the [kubernetes service](https://kubernetes.io/docs/concepts/services-networking/service/) for our application (`'service': {'name': 'hello-kubecon-service'`), as well as the port that service is configured with (`8080`). 
+
+We also see a number of annotations that have been set for the ingress:
+```
+'metadata': {'annotations': {'nginx.ingress.kubernetes.io/proxy-body-size': '20m',
+                             'nginx.ingress.kubernetes.io/rewrite-target': '/',
+                             'nginx.ingress.kubernetes.io/ssl-redirect': 'false'},
+```
+Now let's look at how to change our ingress using Juju configuration.
+## Change configuration
+Looking at the above output we see some default settings for the charm, including `nginx.ingress.kubernetes.io/proxy-body-size': '20m'`. The charm controls this via the [`max-body-size` configuration option](https://charmhub.io/nginx-ingress-integrator/configure#max-body-size). We can easily change this by running:
```

Comment:
s/Looking at the above output we see/In the output above?
Certainly s/above output/output above